tools/tcpconnlat.py

```diff
@@ -44,6 +42,7 @@
 bpf_text = """
 #include <uapi/linux/ptrace.h>
 #include <net/sock.h>
+#include <net/tcp_states.h>
```

Comment:
Without this `tcpconnlat` wouldn't work on our pre-4.6 kernels. I double-checked it still worked under 4.6 with this.